examples/apps/diceroller/webpack.test.cjs

```diff
@@ -35,19 +41,26 @@ module.exports = (env) => {
 			devtoolNamespace: "fluid-example/diceroller",
 			libraryTarget: "umd",
 		},
-		devServer: {
-			static: {
-				directory: path.join(__dirname, "tests"),
-			},
-		},
 		plugins: [
 			new webpack.ProvidePlugin({
 				process: "process/browser.js",
 			}),
 			new HtmlWebpackPlugin({
 				template: "./tests/index.html",
 			}),
+			createExampleDriverServiceWebpackPlugin(service),
 		],
+		devServer: {
+			static: {
+				directory: path.join(__dirname, "tests"),
+			},
+			setupMiddlewares: (middlewares) => {
+				if (service === "odsp") {
+					middlewares.push(...createOdspMiddlewares());
+				}
+				return middlewares;
+			},
```

Comment:
Feels slightly weird to have the full setup for ODSP in the tests. Not blocking on it, but at least I'll throw the thought on the table, of whether it should be here or not.